packages/clay-multi-select/src/index.tsx

```diff
@@ -20,17 +20,33 @@ const ENTER_KEY = 13;
 const DELIMITER_KEYS = [ENTER_KEY, COMMA_KEY];
 
 type Item = {
+	[propName: string]: any;
+
 	/**
 	 * Label to show.
 	 */
-	label: string;
+	label?: string;
 
 	/**
 	 * Hidden value of the item.
 	 */
+	value?: string;
+};
+
+type Locator = {
+	label: string;
 	value: string;
 };
 
+type MenuContentProps = {
```

Comment:
Should this be an interface?

Reply:
As it was just a few simple props, I had only thought to keep `type` but I will switch to `interface` so as not to get away.

Reply:
I think generally we should keep all props as interfaces for the sake of consistency.

Reply:
lol no worries, I actually just caught myself doing that the other day.